packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -286,7 +273,21 @@ class ServiceConnectionManager {
           );
         }
       }
-    }));
+    }
```

Comment:
if we are worried about missing isolate events, what if we only await listening to streams that we are concerned about? perhaps kIsolate and kVM?

Reply:
cc @jacob314 for input on this

Reply:
Ok thought about this a bit more. If the VM is handling our requests in order we don't need to await any of these as long as we ensure that we send the request over the VM Service for the streams before we preform operations getting the current states of the streams. I would double check that we aren't loading isolate objects before the stream listen code and we should be good.

Reply:
I'm not sure if it matters in this case, but there's no guarantee that the VM service will finish processing a request before a more recent request is completed. Obviously this won't be an issue if you're the only client and you're awaiting on each service request, but just something to keep in mind.

Reply:
Confirmed that we are triggering the requests for streams before loading any isolate objects